Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -134,6 +134,61 @@ namespace
         jsInputSource.Set("gripSpace", Napi::External<decltype(inputSource.GripSpace)>::New(env, &inputSource.GripSpace));
     }
 
+    void SetXRHandInputSource(Napi::Object& jsInputSource, xr::System::Session::Frame::InputSource& inputSource)
+    {
+        // Don't set hands up if hand data isn't supported/available
+        if (!inputSource.JointsTrackedThisFrame)
+        {
+            return;
+        }
+
+        constexpr const size_t HAND_JOINT_COUNT{25};
+        constexpr std::array<const char*, HAND_JOINT_COUNT> HAND_JOINT_NAMES{
+            "WRIST",
+
+            "THUMB_METACARPAL",
+            "THUMB_PHALANX_PROXIMAL",
+            "THUMB_PHALANX_DISTAL",
+            "THUMB_PHALANX_TIP",
+
+            "INDEX_METACARPAL",
+            "INDEX_PHALANX_PROXIMAL",
+            "INDEX_PHALANX_INTERMEDIATE",
+            "INDEX_PHALANX_DISTAL",
+            "INDEX_PHALANX_TIP",
+
+            "MIDDLE_METACARPAL",
+            "MIDDLE_PHALANX_PROXIMAL",
+            "MIDDLE_PHALANX_INTERMEDIATE",
+            "MIDDLE_PHALANX_DISTAL",
+            "MIDDLE_PHALANX_TIP",
+
+            "RING_METACARPAL",
+            "RING_PHALANX_PROXIMAL",
+            "RING_PHALANX_INTERMEDIATE",
+            "RING_PHALANX_DISTAL",
+            "RING_PHALANX_TIP",
+
+            "LITTLE_METACARPAL",
+            "LITTLE_PHALANX_PROXIMAL",
+            "LITTLE_PHALANX_INTERMEDIATE",
+            "LITTLE_PHALANX_DISTAL",
+            "LITTLE_PHALANX_TIP"};
```

Comment:
nit: Style. I believe this file is governed by the Babylon Native clang format; if so, you should just be able to run auto-formatting in VS.